### ISSUE TITLE
Setting options on individual translations override, rather than merge global configs

### ISSUE THREAD

**Issue by stiliyan** [i18next/i18next#832]
Passing configurations to individual translations override their global counterparts, when they should be merged instead.

In the following example the desired output is `some <span>foo</span>`, so we set the `interpolation: { escapeValue: false }` option on the translation. Instead `i18next` outputs `some __foo__`, because the global interpolation.prefix and interpolation.suffix configs are overridden, rather than merged with the custom interpolation config.

global config:
```
{
  interpolation: {
    prefix: '__',
    suffix: '__'
  }
}
```

translations:
```
{ 
  "en": {
    "translation": {
      "tag": "some __foo__"
    }
}
```

call with custom options, expecting `some <span>foo</span>` to be output:
```
i18next.t('tag', { interpolation: { escapeValue: false }, foo: '<span>foo</span>' })
```

output:
```
some __foo__
```

**Comment by jamuhl** [comment]
agree....will fix this for next version.

**Comment by jamuhl** [comment]
should be fixed in upcoming 3.5.0

**Comment by jamuhl** [comment]
feel free to reopen if still an issue.
